README.md

```diff
@@ -33,6 +33,36 @@ it with:
 OpenComponents.configure { |config| config.registry = 'http://some.other.host' }
 ```
 
+## Getting Rendered Components
+To request an OpenComponent with its rendered HTML, you can create and load a new
```

Comment:
Is this really passive voice? Wouldn't that be `An OpenComponent with rendered HTML can be requested by creating and loading a new RenderedCompontent object`?